tools/e2e-commons/pages/wp-admin/block-editor.js

```diff
@@ -138,4 +138,12 @@ export default class BlockEditorPage extends WpPage {
 				.map( b => b.name )
 		);
 	}
+
+	async openSettingsSidebar() {
+		const settingsLocator = 'button[aria-label="Settings"][aria-pressed="false"]';
+
+		if ( this.isElementVisible( settingsLocator ) ) {
```

Comment:
A missing `await` here

```suggestion
		if ( await this.isElementVisible( settingsLocator ) ) {
```